<launch>
	<include file="$(find toposens_driver)/launch/serial_interface.launch">
		<arg name="dev_id" value="/dev/ttyUSB0" />
		<arg name="frame_id" value="toposens" />
		<arg name="nWave" value="CnWave00008" />
		<arg name="filtr" value="Cfiltr00003" />
		<arg name="dThre" value="CdThre00010" />
		<arg name="goLim" value="CgoLim-200002000-2000020000001002000" />
	</include>

	<node pkg="toposens_driver" type="markers_publisher" name="markers_publisher" output="screen">
		<param name="frame_id" value="toposens" />
		<param name="lifetime" value="0.5" />
		<param name="volume_thresh" value="40.0" />
		<param name="min_dist" value="0.15" />
		<param name="volume_divider" value="2500.0" />
		<param name="max_volume" value="200.0" />
		<param name="volume_offset" value="0.0" />
		<param name="color_direction" value="z" />
		<param name="color_min" value="0.0" />
		<param name="color_max" value="0.3" />
	</node>

	
	<node pkg="tf" type="static_transform_publisher" name="alpha_link_broadcaster" args="0.049 0 0.112 0 0 0 1 base_link toposens 100" />
	<node pkg="rviz" type="rviz" name="grid_visualization" output="screen" args="-d $(find toposens_driver)/rviz/alpha_markers.rviz" />
	<!--
	
	-->
</launch>
